openedx_events/event_bus/avro/serializer.py

```diff
@@ -42,7 +42,8 @@ def _serialize_non_attrs_values(inst, field, value):  # pylint: disable=unused-a
 
         for extended_class, serializer in all_serializers.items():
             if field:
-                if issubclass(field.type, extended_class):
+                # Make sure that field.type is a class first.
```

Comment:
why is this now specified?

Reply:
In case of `List` types, the check fails if we don't add this check.

![image](https://user-images.githubusercontent.com/10894099/215776016-6997a5d8-d8f7-4e9d-8c8a-48d2646795a9.png)